components/data_police_uk/actions/obtain-crime-outcomes/obtain-crime-outcomes.mjs

```diff
@@ -0,0 +1,54 @@
+import dataPoliceUK from "../../data_police_uk.app.mjs";
+
+export default {
+  key: "data_police_uk-obtain-crime-outcomes",
+  name: "Obtain Crime Outcomes",
+  version: "0.0.1",
+  description: "Access the specific outcomes of reported crimes within a given location and date range. [See the docs here](https://data.police.uk/docs/method/outcomes-at-location/)",
+  type: "action",
+  props: {
+    dataPoliceUK,
+    date: {
+      type: "string",
+      label: "Date",
+      description: "(YYYY-MM) Limit results to a specific month. The latest month will be shown by default",
+      optional: true,
+    },
+    lat: {
+      type: "string",
+      label: "Latitude",
+      description: "Latitude of the requested crime area. You must use either lat/lng or poly, never both.",
+      optional: true,
+    },
+    lng: {
+      type: "string",
+      label: "Longitude",
+      description: "Longitude of the requested crime area. You must use either lat/lng or poly, never both.",
+      optional: true,
+    },
+    poly: {
+      type: "string",
+      label: "Poly",
+      description: "The lat/lng pairs which define the boundary of the custom area. The `poly` parameter is formatted in lat/lng pairs, separated by colons: `[lat],[lng]:[lat],[lng]:[lat],[lng]`.  You must use either lat/lng or poly, never both.",
+      optional: true,
+    },
```

Comment:
It seems that these props are used in multiple actions and sources. Is it possible to move them to the app file?